lib/diary.js

```diff
@@ -0,0 +1,141 @@
+$(document).ready(function(){
+  $.get("https://hidden-shelf-49347.herokuapp.com/api/v1/meals")
+    .then(function(foods){
+      $("h2.left").append(foods[0].name)
+      $("div.left").append($table.addClass("left"))
+      var totalBreakfast = 0
```

Comment:
The numbers are hardcoded right now so that when I load the page the calories are always negative (and in red) but I tested it in my local, and they turn green when there are calories left to be used up in a meal. Thank you for figuring that out.